resources/views/profile/show.blade.php

```diff
@@ -2,7 +2,9 @@
     <div class="flex flex-col items-center justify-center py-10">
         <div class="min-h-screen w-full max-w-md overflow-hidden rounded-lg px-4 dark:shadow-md md:px-0">
             <livewire:links.index :userId="$user->id" />
-            <livewire:questions.create :toId="$user->id" />
```

Comment:
Also make some related changes in question/create component.

For example if I have opened a profile of the user's when he wants to receive questions. Before writing the question and clicking on the send button if that user disables the option I should get a respected message.

Reply:
I have added required validations to question/create.

But for your example I have to poll the database at regular intervals would you be okay with that?

thanks again.

Reply:
Why you wanna poll?

When you just check when someone clicks on the send button?